azavea/fb-gender-survey-dashboard
# Small Features

## Overview

Multiple small changes:

- Update map image to non-repeating image provided by client
- Format Stacked charts to use percentage labels
- Add text denoting 'percentage' values to Grouped and Stacked charts ("Percent who answered..." in the title, and "percent providing given response" on the x-axis)

Connects #64 

### Demo

<img width="1264" alt="Screen Shot 2021-01-25 at 9 48 05 AM" src="https://user-images.githubusercontent.com/21046714/105724074-781b8500-5ef5-11eb-87ff-76df6456ef0c.png">
<img width="989" alt="Screen Shot 2021-01-25 at 9 48 28 AM" src="https://user-images.githubusercontent.com/21046714/105724080-79e54880-5ef5-11eb-9393-c64ff92bd1c9.png">
<img width="1010" alt="Screen Shot 2021-01-25 at 9 48 35 AM" src="https://user-images.githubusercontent.com/21046714/105724083-7baf0c00-5ef5-11eb-92eb-34c09d1b6b26.png">

## Notes

Originally, moving the charts into accordions was included in this set of features; however, @alexelash implemented this in her design pass. 

## Testing Instructions

 * In the Netlify preview, confirm that the map is non-repeating. 
 * Select a country, then select a Stacked, a percentage Grouped, a non-percentage Grouped, and a Waffle question. 
 * Confirm that the percentages show up correctly in the tooltips and the x-axis ticks, for the Stacked chart and the percentage Grouped charts only. 
 * Confirm that the text regarding percentages is applied to the Stacked and percentage Grouped charts only.
